### ISSUE TITLE
Reloading a Post page in Chrome leaves all images as blurred versions

### ISSUE THREAD

**Issue by LorenAmelang** [calebanthony/gridsome-bulma#12]
I kind of suspect this is a Chrome problem, that just appeared recently. I can't believe it was there a couple of months ago and I didn't notice it. 

I just posted the detailed story as a comment to a collection of similar issues at the Gridsome GitHub: 
[https://github.com/gridsome/gridsome/issues/249#issuecomment-623220178](url)

They seem to suspect noscript tags - like the ones that accompany every image in my site. Those seem to be automatically generated - do we have any control over that? 

They also mention "v-html" which is likewise all over the cache and js files, but only two in my files - Post.vue and PostCard.vue, where they are not obviously linked to images. And "v-if" which is in PostCard, plus Author.vue, PostMeta, and Default.vue. Also related to lots of elements which do not seem to be problems. Could those be easily replaced, in case they really are a problem? Way beyond my knowledge...  

I'd appreciate your thoughts!

**Comment by LorenAmelang** [comment]
If you go to 
https://calebanthony.github.io/gridsome-bulma/
it behaves exactly like my site did - refresh works on Index, shows blur image on Post, Index is then blurred and must be refreshed, which fixes Post. 

If you go directly to 
https://calebanthony.github.io/gridsome-bulma/markdown-test-file
Vivaldi and Firefox show the blur image right away and refresh doesn't fix it. For awhile, the Index page did not get corrupted, you could go there and see sharp images, and opening the Post again from the Index page loaded the full image. After several tests of this, Both browsers now show blur images on Index if you load a Post page directly as your first access to the site. 

Somebody is apparently caching some files even through forced reloads! 


Changing g-image to img flips the sort of the Index page, and shows no Post images at all: 
`<img alt="Cover image" src="[object Object]">`

Changing back leaves the sort reversed, but the image blur is fixed! 

Apparently the May 3 updates I installed replaced my modified utils.js file:

C:\Users\loren\Documents\Psychoros\Gridsome Test\bulma\node_modules\gridsome\lib\graphql\nodes\utils.js

```
// 200408LA - changed DESC to ASC:
exports.createSortOptions = function ({ sort, sortBy, order }) {
  if (sort && sort.length) {
    return sort.map(({ by, order }) => [by, order === 'ASC'])
  } else if (sortBy) {
    return [[sortBy, order === 'ASC']]
  }
```
--> Requires Ctrl+C and new Develop!

**BUT somehow the Develop process didn't notice until just now when I changed my Post.vue!** The sort order stayed reversed through tons of Develop and Build sessions over four days! Somehow calling img instead of g-image in Post.vue triggered the system to notice the new utils.js file. 

The problem has been around for awhile - the build I saved before the updates shows it, as does the original starter site which I have no control over. **The js updates have apparently fixed the problem - now that they have been noticed!** 

Is there some trick required when you change certain files, to get them noticed? It looks like everyone now uses Yarn to install updates. This machine has lots of other Node projects, like Node-RED, and all of it has been installed with npm. I tried using Yarn once to update Gridsome, and it destroyed my project. I ended up deleting everything and recreating all my work from scratch. So I'm now afraid to try Yarn on this machine. Maybe things aren't being tested with npm any more? Does the build system use Yarn info to detect updates? 

As for how the image fail happened...

Blurred:

`<img alt="Cover image" src="data:image/svg+xml,%3csvg fill='none' viewBox='0 0 770 70' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'%3e%3cdefs%3e%3cfilter id='__svg-blur-0867bd701f8786bbbbce6cfed6e6dc12'%3e%3cfeGaussianBlur in='SourceGraphic' stdDeviation='1'/%3e%3c/filter%3e%3c/defs%3e%3cimage x='0' y='0' filter='url(%23__svg-blur-0867bd701f8786bbbbce6cfed6e6dc12)' width='770' height='70' xlink:href='data:image/jpeg%3bbase64%2c/9j/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2wBDAQcHBwoIChMKChMoGhYaKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCj/wAARCAAGAEADASIAAhEBAxEB/8QAFwABAQEBAAAAAAAAAAAAAAAAAAUDBv/EACUQAAIBBAIBAwUAAAAAAAAAAAECAwAEESEFEoExQVETMkJh0f/EABcBAAMBAAAAAAAAAAAAAAAAAAIEBQP/xAAhEQACAgIBBAMAAAAAAAAAAAABAgADESEEEhMxQUJh0f/aAAwDAQACEQMRAD8Aq3b3LF1uJVWAsFMcIx2B3gk1b5fjDx1iswdW7EK6417Yx5pSoljEW1oPB8x5tVNZ8tyRFb8lPyMTRTwRFsIBgsAD6nY9a6bkrYWTiOMBlba9vb5/tKURc99V9bmnFYkFydkTmZrdJL4zyjMy5%2bCuBrWtVmOJmuLZoredYY3bP27XO9eaUqjgKuot0C%2b09zepRtuLv7WAfT5N1GfxXGf2axFneQOZkvXLuer9z3DeDSlAiKxJI3C5FK146cjH2f2f/9k=' /%3e%3c/svg%3e" width="770" data-src="/assets/static/histomap-770-70.8253719.9a963660098a26a3c5de51515be7f6c9.jpg" data-srcset="/assets/static/histomap-770-70.8253719.9a963660098a26a3c5de51515be7f6c9.jpg 770w" data-sizes="(max-width: 770px) 100vw, 770px" class="g-image g-image--lazy g-image--loading">`

Working: 

`<img alt="Cover image" src="/assets/static/content/posts/images/histomap-770-70.jpg?width=770&amp;fit=inside&amp;blur=1&amp;key=b2d844e" width="770" data-src="/assets/static/content/posts/images/histomap-770-70.jpg?width=770&amp;fit=inside&amp;blur=1&amp;key=b2d844e" data-srcset="/assets/static/content/posts/images/histomap-770-70.jpg?width=770&amp;fit=inside&amp;blur=1&amp;key=b2d844e 770w" data-sizes="(max-width: 770px) 100vw, 770px" class="g-image g-image--lazy g-image--loaded" srcset="/assets/static/content/posts/images/histomap-770-70.jpg?width=770&amp;fit=inside&amp;blur=1&amp;key=b2d844e 770w" sizes="(max-width: 770px) 100vw, 770px">`

Both have the same noscript section below this.

**Comment by LorenAmelang** [comment]
From @hjvedvik at Gridsome:
@LorenAmelang The problem here is that the post content is wrapped in a paragraph. That will make the generated HTML invalid if it contains any block elements (h1, ul etc.). Changing it to `<div v-html="$page.post.content" />` should help.

@hjvedvik - Thank You! My site was already working properly in "Develop" mode when I posted last, apparently due to the js updates being noticed. But when I actually used "Build", it went back to blurry images in the dist version. Your suggestion to use `<div>` instead of `<p>` seems to have made both versions work! Wonderful birthday present - thanks again! 

--> So the `<p v-html="$page.post.content" />` form is all over the starter source - it wasn't one of my novice hacks. Might fix the starter site to change it to `<div>`...
